### PR TITLE
ARQ-2023: Use relaxed default constructor configuration in Arquillian…

### PR DESCRIPTION
**Pull request by astefanutti**
… Weld embedded adapter for SE environment

### CODE REVIEW

**Comment by tremes** [comment]
Hi @astefanutti I will look at your PRs tomorrow afternoon hopefully and will merge. Thanks a lot!

**Comment by astefanutti** [comment]
@tremes thanks to you!

File: impl/src/main/java/org/jboss/arquillian/container/weld/embedded/mock/FlatDeployment.java [hunk 1]
```diff
@@ -68,6 +71,12 @@ protected void configureServices(Environment environment) {
             services.add(EjbServices.class, new MockEjBServices());
             services.add(SecurityServices.class, new MockSecurityServices());
         }
+        if (environment.equals(Environments.SE)) {
+            ExternalConfigurationBuilder configurationBuilder = new ExternalConfigurationBuilder()
+                // Use relaxed construction by default
+                .add(ConfigurationKey.RELAXED_CONSTRUCTION.get(), true);
```
Comment (johnament) [inline impl/src/main/java/org/jboss/arquillian/container/weld/embedded/mock/FlatDeployment.java]: Is this something we can source from `arquillian.xml`?

Comment (tremes) [inline impl/src/main/java/org/jboss/arquillian/container/weld/embedded/mock/FlatDeployment.java]: These are just common system properties so you can pass them via surefire-plugin.

**Comment by tremes** [comment]
Merged.Thanks!

**Pull request merged**